client/scss/components/_footer.scss

```
@@ -8,7 +8,7 @@
   margin-top: $mobile-nice-padding;
   margin-inline-start: $mobile-nice-padding;
   margin-inline-end: $mobile-nice-padding;
-  z-index: 20;
+  z-index: 32;
```

If we are going to change the z-index, let's pull the value out to the Tailwind config.

`client/tailwind.config.js`
```
      zIndex: {
        'minimap': '80',
        'header': '100',
        'footer-actions': '32',
        'sidebar': '110',
        'sidebar-toggle': '120',
        'dialog': '130',
      },
```

Then use it as a variable like this.

```suggestion
  z-index: theme('zIndex.footer-actions');
```

---

Changed in [9d7746b](https://github.com/wagtail/wagtail/pull/13393/commits/9d7746b41d56324563a02e5d0e102641016c1c33)